src/client/initialize.ts

```diff
@@ -135,7 +136,18 @@ export function initialize<T = {}>(): PluginInstance<T> {
       ) {
         void execPromise('wb:plugin:selection:set', id, elementId, selection);
       },
+      hasTriggerId(id: string) {
+        return registeredActionTriggers.has(id);
+      },
+      triggerAction(id: string) {
+        void execPromise('wb:plugin:action-trigger:invoke', id);
+      },
       configureEditorPanel(options) {
+        registeredActionTriggers.clear();
```

Comment:
This works, but I think we should just access `pluginConfig.config` or `get` directly in `triggerAction` instead of having another object in memory

Reply:
`pluginConfig.config` is not the value of `options`. It is the values that are currently assigned to each option within the slate runtime. We don't currently save the value of `options` anywhere so Alison and I decided this was the best way to handle this.

Reply:
Oh good point, we could just save the config options in a separate object if we wanted, but that doesn't have to be related to this PR

Reply:
Alison and I determined that since we didn't already keep track of config option, then it would be a potential perf loss to start keeping track of all of them if all we needed from them were the action trigger ids